config/routes.rb

```diff
@@ -16,6 +16,8 @@
   get 'tasks', to: 'tasks#my'
   get 'tasks/organisation', to: 'tasks#organisation'
 
+  get 'api/legacy_sync', to: 'sync#legacy'
```

Comment:
still in relation to namespacing API stuff, maybe consider:

```
namespace :api do
  ...
end
```

Reply:
Same